src/clue.js

```diff
@@ -2,27 +2,132 @@
 
 // Suspects Array
 
-const suspectsArray = [];
+const suspectsArray = [
+{
```

Comment:
buen trabajo ten cuidado con las comillas ya que es un objeto  👀 
```suggestion
{
firstName: "Jacob",
lastName: "Green",
occupation: "Entrepreneur",
age: 45,
description: "He has a lot of connections",  // aca esto tambien
image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", // aca esto debe ser un sting 
color: "green",
}
```